File: URI-py/1045.py
# -*- coding: utf-8 -*-
valores = input().split()
valores = list(map(float,valores))
A,B,C = sorted(valores)[::-1]
c = True
if(A >= B+C):
    print("NAO FORMA TRIANGULO")
    c = False
if(A**2 == (B**2) + (C**2) and c):
    print("TRIANGULO RETANGULO")
if(A**2 > (B**2) + (C**2) and c):
    print("TRIANGULO OBTUSANGULO")
if(A**2 < (B**2) + (C**2) and c):
    print("TRIANGULO ACUTANGULO")
if(A == B and B == C and c):
    print("TRIANGULO EQUILATERO")
if((A == B or B == C) and not (A == B and B == C) and c):
    print("TRIANGULO ISOSCELES")

File: URI-cpp/1045.cpp
#include <iostream>
<<<<<<< Updated upstream
#include <iomanip>
#include <vector>
#include <algorithm>
=======
#include <math.h>
>>>>>>> Stashed changes

using namespace std;

int main()
{
<<<<<<< Updated upstream
    double x,a,b,c;
    bool vai = true;
    vector<double>lista;

    for(int i=0;i<3;i++){
        cin >> x;
        lista.push_back(x);
    }

    sort(lista.begin(),lista.begin()+3);
    a = lista[2];
    b = lista[1];
    c = lista[0];

    if(a>=(b+c)){
        cout << "NAO FORMA TRIANGULO" << endl;
        vai = false;
    } else {
        if (a*a==(b*b)+(c*c) && vai) {
            cout << "TRIANGULO RETANGULO" << endl;
        }
        if (a*a>(b*b)+(c*c)  && vai) {
            cout << "TRIANGULO OBTUSANGULO" << endl;
        }
        if (a*a<(b*b)+(c*c)  && vai) {
            cout << "TRIANGULO ACUTANGULO" << endl;
        }
        if (a==b && a==c && vai) {
            cout << "TRIANGULO EQUILATERO" << endl;
        }
        if ((a==b || b==c) && !(a==b && b==c) && vai) {
            cout << "TRIANGULO ISOSCELES" << endl;
        }
    }
=======
    double a,b,c;
    bool Ca = true;

    cin >> a >> b >> c;

    if(a >= b+c){
        cout << "NAO FORMA TRIANGULO" << endl;
        C = false;
    }
    if(a*a==(b*b+c*c)) && Ca){
        cout << "TRIANGULO RETANGULO" << endl;
    }




>>>>>>> Stashed changes
    return 0;
}